test/integration/features/basic.feature

```diff
@@ -7,6 +7,7 @@ Feature: Basic
     Given executing "minishift start --docker-env=FOO=BAR --docker-env=BAZ=BAT"
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
+      And Minishift should have instance config file created
```

Comment:
@hferentschik How about two test case here ? One verify instance config file creation and one verify its content. I am thinking about these two lines:
```
And Minishift should have instance config file created
And Minishift config file should have appropriate content
```